pkg/cmd/pulumi/auth/login.go

```diff
@@ -205,19 +208,29 @@ func NewLoginCmd() *cobra.Command {
 	return cmd
 }
 
-func validateCloudBackendType(typ string) error {
+func validateCloudBackendType(typ string) (bool, error) {
 	kind := strings.SplitN(typ, ":", 2)[0]
 	supportedKinds := []string{"azblob", "gs", "s3", "file", "https", "http"}
 	for _, supportedKind := range supportedKinds {
 		if kind == supportedKind {
-			return nil
+			if strings.Contains(kind, "http") {
+				return true, nil
+			}
+			return false, nil
 		}
 	}
-	return fmt.Errorf("unknown backend cloudUrl format '%s' (supported Url formats are: "+
+	return false, fmt.Errorf("unknown backend cloudUrl format '%s' (supported Url formats are: "+
 		"azblob://, gs://, s3://, file://, https:// and http://)",
 		kind)
 }
 
+func checkHTTPCloudBackendURL(url string) string {
+	if strings.HasSuffix(strings.TrimSuffix(url, "/"), ".pulumi.com") {
```

Comment:
```suggestion
	if strings.HasSuffix(strings.TrimSuffix(url, "/"), "app.pulumi.com") {
```

Maybe let's make it more localized? Not sure it should go to `api.pulumi.com` if I do `pulumi login abc.pulumi.com`

Reply:
Hi @tgummerer , I have made it more restrictive.
Thanks!